src/package_io/parse_input.star

```diff
@@ -100,10 +103,13 @@ def parse_input(input_args):
 	if result["network_params"]["deneb_fork_epoch"] == 0:
 		fail("deneb_fork_epoch is 0 needs to be > 0 ")
 
-	required_num_validtors = 2 * result["network_params"]["slots_per_epoch"]
-	actual_num_validators = len(result["participants"]) * result["network_params"]["num_validator_keys_per_node"]
-	if required_num_validtors > actual_num_validators:
-		fail("required_num_validtors - {0} is greater than actual_num_validators - {1}".format(required_num_validtors, actual_num_validators))
+	if total_participant_count < 1:
```

Comment:
I don't particularly see a scenario where total_participant_count could be less than 1. If there are no participants, there wouldn't be any test running at all.